### PR TITLE
Fix monkey, release no longer need a real device

### PR DESCRIPTION
**Pull request by nhachicha**
As reported by @beeender https://github.com/novoda/gradle-android-command-plugin/issues/122 the `android-command` plugin has an issue when running `monkey`. This PR remove this dependency, also we no longer require having a physical device to run monkey task (and perform a release)

fixes https://github.com/realm/realm-java/issues/5192

### CODE REVIEW

**Comment by nhachicha** [comment]
@realm/java

File: examples/build.gradle [hunk 1]
```diff
@@ -38,6 +46,29 @@ allprojects {
         jcenter()
         google()
     }
+
+    if (!project.name.startsWith("realm-examples")
+            && !project.name.startsWith("library")
+            && !project.name.startsWith("moduleExample")) { // exclude root and library project
+        task runMonkey(dependsOn: 'installDebug') {
```
Comment (beeender) [inline examples/build.gradle]: we still need to run the `monkeyRelease` after uploading to bintrary. This doesn't seem to handle it?

Comment (beeender) [reply examples/build.gradle]: I don't quite understand this, the task to run monkey for example is still `:monkeyDebug` and `:monkeyRelease`?

Comment (nhachicha) [reply examples/build.gradle]: updated the script, now it has the same name as the previous tasks, also generating a release & debug tasks

**Pull request merged**